lib/fog/metering/openstack/requests/get_samples.rb

```diff
@@ -2,7 +2,7 @@ module Fog
   module Metering
     class OpenStack
       class Real
-        def get_samples(meter_id, options = [])
+        def get_samples(meter_id, options = [], limit = 10000000)
```

Comment:
No - it sets the `data` hash with `q` and then JSON encodes it as the HTTP post body.